demos/common/cpp/utils/include/utils/common.hpp

```diff
@@ -365,3 +220,31 @@ inline std::string fileNameNoExt(const std::string &filepath) {
     if (pos == std::string::npos) return filepath;
     return filepath.substr(0, pos);
 }
+
+inline void printExecNetworkInfo(const InferenceEngine::ExecutableNetwork& execNetwork, const std::string& modelName,
```

Comment:
You should align on on implementations of `printExecNetworkInfo()` and `log_runtime_settings()` with @anzhella-pankratova.

Reply:
we aligned the way nstreams, threads and infer requests are reported [(python impl)](https://github.com/openvinotoolkit/open_model_zoo/pull/2567/commits/a775b5d3d7c59cca9a70e41af430a629b522915e). Is it ok right now?

Reply:
Yes